lte/gateway/Makefile

```diff
@@ -207,6 +207,17 @@ test_c: ## Run all Bazel-ified C/C++ tests
 test_oai: ## Run all OAI-specific tests
 	$(call run_ctest, $(C_BUILD)/core, $(C_BUILD)/core/oai, $(GATEWAY_C_DIR)/core, $(OAI_FLAGS) $(OAI_TEST_FLAGS) $(OAI_NOBENCHMARK_FLAGS), $(OAI_TESTS))
 
+test_oai_runtime: export REPORT_FOLDER=${MAGMA_ROOT}/report/
+test_oai_runtime: export UNITTEST_REPORT_FOLDER=${REPORT_FOLDER}/unittest_report/
+test_oai_runtime: export MERGED_REPORT_FOLDER=${REPORT_FOLDER}/merged_report/
+test_oai_runtime: export GTEST_OUTPUT=xml:${UNITTEST_REPORT_FOLDER}
```

Comment:
What is the reason of setting these under `test_oai_runtime`? We should set these at the top of the file as the other variables

Reply:
Hi Alex, my purpose is to limit the effect only inside the `test_oai_runtime`. If we put it at the top of the file, it might affect other targets.